app/src/main/res/layout/activity_main.xml

```diff
@@ -66,6 +66,31 @@
         app:layout_constraintTop_toBottomOf="@id/btn_set_notification_time" />
 
 
+    <RadioGroup
+        android:id="@+id/rg_single_repeat"
+        app:layout_constraintTop_toBottomOf="@id/tv_alarm_date"
+
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal">
+
+        <RadioButton
+            android:id="@+id/rb_single_alarm"
+        android:layout_width="wrap_content"
```

Comment:
Nit: the whitespace is a bit off on these two lines.  Does Android Studio provide a way to auto-format XML files?  If so, we should let it do its thing.

Reply:
Maybe? I'd really like it to, but it doesn't do it automatically. I haven't felt like exploring the 20,000 menu options, but my life would be easier if it would fix things like this. I'll poke around a bit.